core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -681,7 +715,13 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
     self.zipWith(that)((a, b) => (a, b))
 
   /**
-   * A named alias for `&&&`.
+   * Alias for `&&&`.
+   */
+  final def <*>[R1 <: R, E1 >: E, B](that: ZIO[R1, E1, B]): ZIO[R1, E1, (A, B)] =
+    self &&& that
+
+  /**
+   * A named alias for `&&&` or `<*>`.
```

Comment:
I think we can delete `&&&` now that we have `<*>`. I know Haskell uses `&&&` but no reason for two symbols for the same operator.